articulation_msgs/ParamMsg[] params
std_msgs/Duration target_duration
---
std_msgs/Bool success
std_msgs/String error_message
